### PR TITLE
Implement Blobovnicza

### PR DESCRIPTION
**Pull request by cthulhu-rider**
## Plan
- [x] Define type

- [x] Add config values

- [x] Implement methods (Put, Get, Delete, Open, Close, Init)

- [x] Write unit tests

### CODE REVIEW

File: pkg/local_object_storage/blobovnicza/sizes.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+	return bucketKeyFromBounds(upperBound)
+}
+
+func (b *Blobovnicza) incSize(sz uint64) {
+	b.filled.Add(sz)
+}
+
+func (b *Blobovnicza) decSize(sz uint64) {
+	b.filled.Add(sz)
```
Comment (cthulhu-rider) [inline pkg/local_object_storage/blobovnicza/sizes.go:47]: `b.filled.Dec(sz)`

File: pkg/local_object_storage/blobovnicza/blobovnicza.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+	log *logger.Logger
+}
+
+type boltDBCfg struct {
+	perm os.FileMode
+
+	path string
+
+	boltOptions *bbolt.Options
```
Comment (alexvanin) [inline pkg/local_object_storage/blobovnicza/blobovnicza.go:39]: `boltOptions` has no set references in the code. Is it correct?

File: pkg/local_object_storage/blobstor/put.go [hunk 3]
```diff
@@ -61,12 +61,6 @@ func (t *fsTree) put(addr *objectSDK.Address, data []byte) error {
...
-func (b *BlobStor) isBig(obj *object.Object) bool {
-	// FIXME: headers are temporary ignored
-	//  due to slight impact in size.
-	//  In fact, we have to honestly calculate the size
-	//  in binary format, which requires Size() method.
-	sz := obj.PayloadSize()
-
-	return sz > b.smallSizeLimit
+func (b *BlobStor) isBig(obj *object.Object, data []byte) bool {
+	return uint64(len(data)) > b.smallSizeLimit
```
Comment (alexvanin) [inline pkg/local_object_storage/blobstor/put.go:65]: Do we need object argument there?

File: pkg/local_object_storage/blobovnicza/blobovnicza.go (hunk 2 above)
Comment (cthulhu-rider) [reply pkg/local_object_storage/blobovnicza/blobovnicza.go:39]: I planned to add option to specify DB options, but forgot, thx.

File: pkg/local_object_storage/blobstor/put.go (hunk 3 above)
Comment (cthulhu-rider) [reply pkg/local_object_storage/blobstor/put.go:65]: No longer needed

File: pkg/local_object_storage/blobovnicza/delete.go [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+// SetAddress sets address of the requested object.
+func (p *DeletePrm) SetAddress(addr *objectSDK.Address) {
+	p.addr = addr
+}
+
+// Delete removes object from Blobovnicza by address.
+//
+// Returns any error encountered that
+// did not allow to completely delete the object.
+func (b *Blobovnicza) Delete(prm *DeletePrm) (*GetRes, error) {
```
Comment (cthulhu-rider) [inline pkg/local_object_storage/blobovnicza/delete.go:27]: `GetRes` -> `DeleteRes`

**Pull request merged**